turbo/cli/flags.go

```diff
@@ -292,6 +293,9 @@ func ApplyFlagsForEthConfig(ctx *cli.Context, cfg *ethconfig.Config, logger log.
 		}
 		etl.BufferOptimalSize = *size
 	}
+	prune := ctx.String(PruneFlag.Name)
+	cfg.SnapshotPrune = strings.Contains(prune, "h") // Pruning of snapshots is enabled with h.
+	cfg.SnapshotsPruneBlockNumber = ctx.Uint(PruneHistoryBeforeFlag.Name)
```

Comment:
btw just double checking whether this should be `PruneHistoryFlag.Name` instead since that is for "Prune data older than this number of blocks from the tip" while `PruneHistoryBeforeFlag` is for "Prune data before this block"?

Reply:
it just says that block's pruning is tied to history